### DISCUSSION TITLE
Problems with the definition of "disability"

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [CommonCoreOntology/CommonCoreOntologies#344] [Announcements]
The definition of "disability" reads:

> A Quality inhering in an Agent by virtue of a physical or mental condition that limits the Agent's movements, senses, or activities.

Does any physical or mental condition of the relevant sort count here, or only those that are internal/intrinsic to the Agent?

If the former, then it seems the definition would count an agent who is in the condition (i.e., state) of being tied up or being blindfolded as having a disability, which is clearly mistaken.

However, even if the latter, problems may remain. I am in the internal/intrinsic condition of being two-legged, and that condition limits my movements in certain ways, yet I do not have a disability by virtue of being in that condition. Similarly, I am in the internal/intrinsic condition of having a certain eye structure, and that condition limits my senses (by, e.g., making me unable to see in the ultraviolet range), yet (again) I do not have a disability by virtue of having that eye structure.

**Comment by cameronmore** [comment]
[This paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4815270/) (partially by Alex Cox @APCox  and Mark Jensen @mark-jensen ) goes a long way in addressing these concerns, and 'blindness' can be seen as a analogue for other kinds of 'disabilities'.

Some disabilities are temporary, some permanent, and even what counts as a disability can vary from context to context, but there is a rich area here to explore.

The current term, as you have pointed out, is agnostic on many of these issues to allow users to be flexible with what more specific kind of disability they are modeling. In many cases, the 'gain' or 'loss' of disposition classes may be useful. However, the process of losing a disposition itself is not a quality, and the term 'disability' would refer to that process, and therefore should be thought of as a kind of measurement. (of some process having occurred). So, by calling disability a quality implicitly references this past un-disabled state, and goes into measurement territory.

There are many other things to consider, but those come to mind first.

In any event, this term can be improved upon with some some clarification and better design patterns.

**Comment by gregfowlerphd** [comment]
@cameronmore:

Thanks for the paper! I'll give it a look. :)

You might be being too generous when you say that I pointed out the flexibility of the definition. I was simply putting forward some purported counterexamples to it.

I do agree that such flexibility is desirable. However, counterexamples are, presumably, undesirable. Although perhaps you meant to deny that the cases I presented constitute counterexamples? Sorry, that wasn't entirely clear to me.

**Comment by gregfowlerphd** [comment]
@cameronmore:

I finally got a chance to read the paper on blindness you linked to. Thanks again for referring me to it. I found it interesting and enjoyable, and something like the proffered account of blindness seems promising. Perhaps, as I think you were suggesting, something like it, suitably generalized, could be used to formulate a more satisfactory replacement for the current definition of 'disability'.

[Despite being a fan of the paper, I do have some criticisms of it. For one thing, I think the authors are mistaken when they claim that in BFO, a realizable entity cannot undergo a decrease. That claim certainly conflicts with the definitions of 'damaged stasis' and 'wounded stasis' in the Event Ontology, and I believe it also conflicts with some of what Barry Smith has said in his writings on realizable entities, though I'd have to go back and reread them to be sure.

For another, I don't think the proffered account of blindness--according to which "blindness is a reduction of the trigger conditions under which the sight function is realized"--can be quite right. (Hence why I said only that _something like it_ seems promising.) Here are two worries about it:

- It seems natural to treat a reduction as a process during which something is reduced, a process which ends when the reducing stops. But if that's right, then the proffered account gets the temporal facts about blindness wrong: By identifying blindness with the relevant reduction process, it implies that someone's blindness ceases when that process ends. In fact, however, someone's blindness will typically outlast the reduction process.
- The proffered account seems unable to handle certain clear-cut cases of blindness. Consider, in particular, people with congenital blindness whose sight is _always_ limited to the same degree and hence never experience a reduction of the relevant sort. The proffered account seems to imply, incorrectly, that these people are not blind.]

**Comment by cameronmore** [comment]
> the authors are mistaken when they claim that in BFO, a realizable entity cannot undergo a decrease

As I understand, this is not quite a settled issue, and the Common Core Ontologies offer an account of realization and disposition that extends BFO, but does not (aim to) contradict it. BFO is agnostic with respect to the representation of actuality v. possibility, for example, and the CCO offers one, of multiple, interpretations with the modal relation ontology.

> It seems natural to treat a reduction as a process during which something is reduced, a process which ends when the reducing stops. But if that's right, then the proffered account gets the temporal facts about blindness wrong: By identifying blindness with the relevant reduction process, it implies that someone's blindness ceases when that process ends. In fact, however, someone's blindness will typically outlast the reduction process.

Consider realization to be a formula of disposition + stimulus + context = realization, where realization causes some effect. For example, my hand has the disposition to grip something, and when it realizes this disposition, something in the world is actually gripped (a change in a given state of affairs has occurred).

Your response conflates parts of these activities of reduction and being-reduced. There is a process which reduces the ability for the triggering stimulus (light) to reach the relevant parts of the eye, and that process has the result that the disposition to absorb light and see is reduced. The effect of the reduction can outlast the process of reducing--there's no problem with that from the BFO perspective. In the paper, they offer an account of a door that has been altered and is therefore more difficult to open. Such a scenario is like the damaging of certain portions of an eye. The eye has more difficulty in realizing its function because of some damage that occurred. Some damage may be permanent, some damage may be temporary, some damage may make physical changes that only take an instant--like staring at an eclipse for a few seconds--but lifetime length impacts. I don't see the paper making the error you are pointing toward.

> The proffered account seems unable to handle certain clear-cut cases of blindness. Consider, in particular, people with congenital blindness whose sight is always limited to the same degree and hence never experience a reduction of the relevant sort. The proffered account seems to imply, incorrectly, that these people are not blind.

Well, sort of. Whether one is born without the ability to see has no bearing on the fact that the eye _does in fact_ bear that function, it just never may be realized because of some injury or defect.

**Comment by cameronmore** [comment]
I think fully expanding out the ideas in the paper would be useful. Do you have a suggested definition we could replace the current one with? If you offer a start, we can wordsmith something here, but the paper is certainly a good start as you mentioned.

**Comment by gregfowlerphd** [comment]
@cameronmore:

Let me start by addressing the replies you give in your first comment to my three criticisms of the paper:

- _First criticism_: I think we basically agree here: BFO is neutral about whether a realizable entity can undergo a decrease. Indeed, my criticism was that the paper didn't take BFO to be neutral, but instead took BFO to entail that a realizable entity cannot undergo a decrease.
- _Second criticism_: I agree with the distinctions you draw here (although I'm not sure which of the things you distinguish between you think I conflated). I'm pretty sure I also agree with your other claims about the mechanism behind the onset of blindness. But I'm not convinced by your reply. You note, correctly, that "the _effect_ of the reduction can outlast the process of reducing" (my italics). However, the proffered account of blindness--i.e., "blindness is a reduction of the trigger conditions under which the sight function is realized"--identifies blindness with the _reduction itself_, not with its _effect_. It thus implies that blindness lasts only as long as the reduction lasts, regardless of whether its effect lasts longer. (Of course, an account which _instead_ identified blindness with the effect of the reduction would have different, and perhaps more plausible, implications concerning the temporal facts about blindness.)
- Third criticism: I admit I'm puzzled by your reply here. You point out that in the cases of congenital blindness I describe, the eye still bears the function to see. But I never denied that the function exists in those cases (and, in fact, I agree that it does). What I did deny is that "a reduction of the trigger conditions under which the sight function is realized" exists in those cases (because in those cases, the trigger conditions remain constant for the person's entire life). Since the proffered account identifies blindness with such a reduction, it implies that no blindness exists in those cases.

In your second comment, you ask whether I have a suggested replacement definition for 'disability'. I wish that I did, but I suspect that constructing a satisfactory definition would be a very tough task. I will say this, though: I'm inclined to think that a satisfactory definition would involve the notion of canonicity, a la the Foundational Model of Anatomy.

Your request for a replacement definition did raise a question for me, though. In my ongoing exploration of CCO, I've noticed that every class has a definition. Why is this? Don't get me wrong; I understand that it's best to provide a definition when possible. But in the case of terms like 'disability', where a satisfactory definition is very difficult to construct, might it not be better to provide an elucidation, along with some clarificatory comments, instead?

**Comment by cameronmore** [comment]
Perhaps we disagree with how to read this line:

`Blindness is a reduction of the trigger conditions under which the sight function is realized`

I don't think this identifies blindness with the reduction, but blindness is the effect of reduction. On the one hand, I see your interpretation, but I am a little more generous because the examples given throughout the paper don't lead me to think that the paper is making that error. This could be be clarified by distinguishing 'blindness' from 'being blind'. In that case, blindness could be the reduction and the 'being blind' could be the enduring state of not being able to realize the disposition of sight.

It is not a good idea to leave classes undefined since users can interpret these however they like which may lead to inconsistent and incompatible models, which defeats the purpose of having a standard model and design pattern. But, some terms are primitives, mostly at the BFO level, which only have elucidations and comments, rather than formal definitions.

In the case of this term, we should definitely provide other annotations like elucidations and comments to give more clarity for what a term means and the normative things that the term intends to represent.

**Comment by gregfowlerphd** [comment]
@cameronmore:

Fair enough re: the paper on modeling blindness. While the line in question reads to me as a straightforward identification, you might be right that that wasn't the intent.

Thanks for your answer to my question about definitions vs. elucidations and comments. :)

**Comment by cameronmore** [comment]
I think the term vulnerability might already capture the notion of disability discussed above. 

Vulnerability = A Disrupting Disposition the realization of which would disrupt a process that the bearer of the Disrupting Disposition has an interest in.

(And , Disrupting Disposition = A disposition the realization of which would disrupt a process some entity has an interest in.)

The bearer of a disability is disrupted when that disability disrupts and interferes with processes they participate in. A disability sounds like a vulnerability with some other criteria which we can still think on, but I think this is a good direction.

**Comment by gregfowlerphd** [comment]
@cameronmore: This is an interesting proposal, and I agree with you that it's potentially promising. One question to consider might be: Should the disability be identified with the vulnerability itself--a disposition--or rather with the internal physical grounds of this disposition--a quality?

**Comment by cameronmore** [comment]
We should work offline on it a bit, glad you agree

**Comment by oliviahobai** [comment]
Moving issue to discussion thread to continue development
